### ISSUE TITLE
Support for warp?

### ISSUE THREAD

**Issue by alex-s-gardner** [evetion/GeoArrays.jl#123]
Are there any intentions for GeoArrays to support warping between projections?

**Comment by evetion** [comment]
Do you have a specific form of warp in mind? Only a vertical change, or creating completely new geoarrays? Maybe warp from one GeoArray into another one? That would help with designing an api.

The options for warping in pure Julia are limited, but we should be easy to call GDAL for this.

**Comment by alex-s-gardner** [comment]
@evetion I've been using the interpolations.jl which seems pretty fast. The one thing that I do a lot is interpolate (warp) one array into the same extent, projection and resolution as another array so they can be operated on together.

**Comment by evetion** [comment]
Interpolations.jl might be nice for a package extension. I've implemented gdalwarp in version 0.8.

```julia
ga = GeoArray(zeros((360, 180)))
bbox!(ga, (min_x=-180, min_y=-90, max_x=180, max_y=90))
crs!(ga, GeoFormatTypes.EPSG(9754))
ga2 = GeoArrays.warp(ga, Dict("t_srs" => "EPSG:4326+3855"))
```

There's also `warp(ga, like::GeoArray)`, which retrieves the crs, extent and size from the `like` GeoArray.

Arguments in the Dict are described in https://gdal.org/programs/gdalwarp.html
